fury/ui/helpers.py

```diff
@@ -129,3 +123,15 @@ def check_overflow(textblock, width, overflow_postfix='',
             if side == 'left':
                 textblock.message = textblock.message[::-1]
             return mid_ptr
+
+
+def is_ui(actor):
```

Comment:
you can put this in `fury.utils` for now.

Reply:
Done.